README.md

```diff
@@ -4,13 +4,19 @@
 
 ## Workflow
 
-git checkout -b "branch_name"
-git add .
-git commit -m"description"
-git pull origin "branch_name"
-github steps
-git checkout main
-git pull origin main
+### Github Steps
+
```

Comment:
there should not be any spaces between list elements

are you sure you are using prettier to format this on save?

make sure prettier is enabled for markdown files, in the bottom blue bar